### PR TITLE
New package: pulsemixer-1.3.0

### PR DESCRIPTION
**Pull request by z1lt0id**
Re-submitting with vlicense. Had to resubmit due to corrupt repo.

### CODE REVIEW

File: srcpkgs/fuseiso/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'fuseiso'
+pkgname=fuseiso
+version=20070708
+revision=1
+only_for_archs="i686 x86_64"
```
Comment (Vaelatern) [inline srcpkgs/fuseiso/template]: no.

File: srcpkgs/fuseiso/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+build_style=gnu-configure
+configure_args="--prefix=/usr"
```
Comment (Vaelatern) [inline srcpkgs/fuseiso/template]: no

File: srcpkgs/fuseiso/template [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+patch_args="-Np1"
+makedepends="pkg-config fuse-devel zlib-devel glib-devel"
```
Comment (Vaelatern) [inline srcpkgs/fuseiso/template]: no: `makedepends="fuse-devel zlib-devel glib-devel"` and `hostmakedepends="pkg-config"`

File: srcpkgs/fuseiso/template [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+depends="fuse glib zlib"
```
Comment (Vaelatern) [inline srcpkgs/fuseiso/template]: no

File: srcpkgs/fuseiso/template [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+license="GPL-2"
+homepage="http://sourceforge.net/projects/fuseiso"
+distfiles="http://ubiz.ru/dm/${pkgname}-${version}.tar.bz2"
+checksum=8b242e077d66cd20900c59c905ff90b4c934b0613dd5a20facb0b1260ac5fd88
+
+do_install() {
+	vbin src/fuseiso
+}
+
+
```
Comment (Vaelatern) [inline srcpkgs/fuseiso/template]: spare newlines?

File: srcpkgs/fuseiso/template [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+depends="fuse glib zlib"
+short_desc="FUSE module to mount ISO filesystem images"
+maintainer="fs0ciety <mrrobot@fsociety.info>"
+license="GPL-2"
+homepage="http://sourceforge.net/projects/fuseiso"
+distfiles="http://ubiz.ru/dm/${pkgname}-${version}.tar.bz2"
+checksum=8b242e077d66cd20900c59c905ff90b4c934b0613dd5a20facb0b1260ac5fd88
+
+do_install() {
+	vbin src/fuseiso
```
Comment (Vaelatern) [inline srcpkgs/fuseiso/template]: Is there no `make install`?

File: srcpkgs/pulsemixer/template [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+distfiles="https://fsociety.info/packages/pulsemixer/${version}.tar.gz"
+checksum=3e484f10a0c07324324f7ec3f484fb88628ccd661c07728828060a44d07223b9
+
+post_install() {
+	vlicense LICENSE
+}
+
+
+
+
```
Comment (Vaelatern) [inline srcpkgs/pulsemixer/template]: spare newlines at the end

**Comment by Vaelatern** [comment]
Welcome to using git. Please split these into two different branches. Do not close this PR. Once you have two branches, and only then, push them to your github account. This PR is on your master branch, so leave the pulsemixer package on this branch.

You did not need to close and re-open due to anything. You just don't yet know how to use git, but that makes working with you to fix your templates a lot harder. Please learn git by doing the above exersize. Though you may need to totally remove your local git repository in the process, at no point will you need to close this PR.

Thank you, and I look forward to adding these packages to our repository.

**Comment by Vaelatern** [comment]
The fuseio package patches are okay only due to the age of the software and time since the last change (upstream seems dead). I think they can stay. Whether we accept the fuseio package or not is a different question.

File: srcpkgs/pulsemixer/template [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+revision=1
+noarch=yes
+build_style=python3-module
+hostmakedepends="python3 python3-setuptools"
+depends="pulseaudio"
+short_desc="CLI and curses mixer for Pulseaudio"
+maintainer="Dean Thomson <mrrobot@fsociety.info>"
+license="MIT"
+homepage="https://github.com/GeorgeFilipkin/pulsemixer"
+distfiles="https://fsociety.info/packages/pulsemixer/${version}.tar.gz"
```
Comment (Vaelatern) [inline srcpkgs/pulsemixer/template]: No. This is not upstream or an upstream source. This is your own source that happens to add a file upstream has not added. The correct way to handle this is to tell upstream that they need to include the text of the MIT license to be licensed under its terms (according to my understanding) and would they please add such a copy to their repo.

Comment (Vaelatern) [reply srcpkgs/pulsemixer/template]: Everything else notwithstanding, this is a massive blocker.

Comment (z1lt0id) [reply srcpkgs/pulsemixer/template]: I got the author to put a MIT License file in his git repo and it is now in the latest 1.3.0-release tar file on his project page.  I'll fix this package up now - at this point I'm not sure I'll continue with fuseiso.

**Comment by asergi** [comment]
c0dcec00bf77dc71333d2e52bdd2d95a7f66d28e

**Pull request closed without merging**